extensions/simplelogin/src/list_alias.tsx

```diff
@@ -60,6 +65,7 @@ export default function Command() {
         title: "Are you sure?",
         message: "Do you really want to delete this alias? This action cannot be undone.",
         icon: Icon.DeleteDocument,
+        rememberUserChoice: true,
```

Comment:
I tested this, and if I dismiss the alert and chech the box, I won't be prompted for deletion again.
We probably want this behavior only  when someone decides to go ahead with deletion?

Reply:
That's the way it works. If you first tick the "don't show again," it won't pop up again